pyproject.toml

```diff
@@ -5,7 +5,7 @@ build-backend = "poetry.core.masonry.api"
 
 [tool.poetry]
 name = "sparrow-datums"
-version = "0.7.2"
+version = "0.8.0"
```

Comment:
It's best practice to increase the minor version instead of patch when the interface changes